components/brave_ads/core/internal/account/utility/redeem_confirmation/non_reward/redeem_non_reward_confirmation.cc

```diff
@@ -86,7 +86,7 @@ void RedeemNonRewardConfirmation::CreateConfirmationCallback(
   BLOG(6, UrlResponseToString(mojom_url_response));
   BLOG(7, UrlResponseHeadersToString(mojom_url_response));
 
-  if (mojom_url_response.code != net::HTTP_NO_CONTENT) {
+  if (mojom_url_response.code < 200 || mojom_url_response.code > 399) {
```

Comment:
I don't think we want to support redirects here.

Maybe we just need to allow `200` and `204`?

Reply:
Changed, thanks